src/Uno.UI.RuntimeTests/Tests/Windows_UI_Xaml_Controls/Given_TextBox.cs

```diff
@@ -473,5 +474,29 @@ public async Task When_SelectedText_AllTextToEmpty()
 			Assert.AreEqual(0, textBox.SelectionStart);
 			Assert.AreEqual(0, textBox.SelectionLength);
 		}
+
+		[TestMethod]
+		public async Task When_Text_Overflows()
+		{
+			// iOS used to produce ellipsis when text overflows.
+			// We make the text overflow with spaces (has no color),
+			// then we assert that we don't have red pixels in the textbox.
+			var SUT = new TextBox
+			{
+				Text = new string(' ', 100),
+				Width = 100,
+				Foreground = Colors.Red,
+			};
+
+			WindowHelper.WindowContent = SUT;
+			await WindowHelper.WaitForIdle();
+			var renderer = new RenderTargetBitmap();
+			await renderer.RenderAsync(SUT);
+			var si = new RawBitmap(renderer, SUT);
+			await si.Populate();
+
+			ImageAssert.DoesNotHaveColorInRectangle(si, new System.Drawing.Rectangle(new System.Drawing.Point(0, 0), si.Size), Colors.Red, tolerance: 20);
+		}
```

Comment:
This test is already passing on iOS without the fix (see https://github.com/unoplatform/uno/pull/9684)

This needs some investigation from someone who has a Mac